mcs/class/System.Security/System.Security.Cryptography.Xml/corefx/SR.cs

```diff
@@ -0,0 +1,96 @@
+//
```

Comment:
.NET core code has identifiers with same names so that's why we use common corefx/ in root where resx2sr can handle the duplicates for us.